app/services/check_answers/sections/assets.rb

```diff
@@ -0,0 +1,135 @@
+        #              - attribute: valuables_relevant
+        #                type: boolean
+        #              - attribute: valuables
+        #                type: money
+        #                disputed_if: valuables_in_dispute
+        #                skip_unless: valuables_relevant?
+      end
+
+      def partner_asset_tables
+        #      - tables:
+        #          - screen: partner_assets
+        #            fields:
+        #              - many: true
+        #                model: partner_bank_accounts
+        #                template:
+        #                  attribute: amount
+        #                  type: money
+        #              - attribute: partner_investments_relevant
+        #                type: boolean
+        #              - attribute: partner_investments
+        #                type: money
+        #                skip_unless: partner_investments_relevant?
+        #              - attribute: partner_valuables_relevant
+        #                type: boolean
+        #              - attribute: partner_valuables
+        #                type: money
+        #                skip_unless: partner_valuables_relevant?
+        asset_tables(screen: :partner_assets, attribute_prefix: "partner_", investments_disputed: false, valuables_disputed: false)
+      end
+
+      def asset_tables(screen:, attribute_prefix:, investments_disputed:, valuables_disputed:)
+        accounts = @check.public_send("#{attribute_prefix}bank_accounts") || []
+        bank_fields = accounts.map.with_index do |account, index|
+          MoneySubFieldPresenter.new(table_label: screen, attribute: :amount, index: index + 1, model: account, disputed: account.account_in_dispute)
+        end
+
+        table = Table.new(screen:, skip_change_link: false, index: nil, disputed?: nil,
+                          fields:
+                            bank_fields +
+                            [
+                              FieldPresenter.new(table_label: screen, attribute: "#{attribute_prefix}investments_relevant".to_sym, type: :boolean, model: @check),
+                              if @check.public_send("#{attribute_prefix}investments_relevant?".to_sym)
+                                MoneyPresenter.new(table_label: screen, attribute: "#{attribute_prefix}investments".to_sym, model: @check, disputed: investments_disputed)
+                              end,
+                              FieldPresenter.new(table_label: screen, attribute: "#{attribute_prefix}valuables_relevant".to_sym, type: :boolean, model: @check),
+                              if @check.public_send("#{attribute_prefix}valuables_relevant?".to_sym)
+                                MoneyPresenter.new(table_label: screen, attribute: "#{attribute_prefix}valuables".to_sym, model: @check, disputed: valuables_disputed)
+                              end,
+                            ].compact)
+        [table]
+      end
+
+      def vehicle_tables
```

Comment:
Do we need a condition to only show this on a certificated check? I get this additional table (marked in red) on a controlled check. 

![Screenshot 2024-08-12 at 13 39 29](https://github.com/user-attachments/assets/07ca4411-8ba9-4a5c-b023-9d5175e140f6)